src/main/resources/typescript-axios-webpack-module-federation/api.mustache

```diff
@@ -0,0 +1,13 @@
+/* tslint:disable */
+/* eslint-disable */
+
+import { APIFactory } from '@redhat-cloud-services/javascript-clients-shared/dist/utils';
+import type { AxiosInstance } from 'axios';
+import {
+{{>endpointList}}  } from './index';
+
+export const client = (BASE_PATH: string, instance: AxiosInstance) => {
```

Comment:
Can we change the export name to something like
```tsx
export const HostInventoryClient
```
or similar? Otherwise we will have a naming conflict importing 2 clients at the same time.